src/index.js

```diff
@@ -1024,7 +1024,7 @@ export default class MeasureTool {
       this._nodeText
         .select(':last-child')
         .text(
-          `Total distance: ${this.lengthText}; Total area: ${this.areaText}.`
+          Config.distanceAndLength(this._options.language)(this.lengthText, this.areaText)
```

Comment:
this will break the language where it doesn't have this configurations, what do you think of a fallback?

Reply:
Can add a fallback to default language (english), nice to you?

Reply:
fallback as english sounds good